docs/pages/index.mdx

```diff
@@ -1,221 +1,124 @@
+  cryptographic identity.
+- [Set up passwordless authentication](access-controls/guides/passwordless.mdx)
+  to enable users to access resources with hardware keys, including biometric
+  credentials like Touch ID and YubiKey Bio.
+- [Integrate your Single Sign-On provider](access-controls/sso.mdx): Allow users
+  to access infrastructure resources with IdPs like Okta.
+- [Use Teleport as an identity provider](access-controls/idps/saml-guide.mdx) to
+  authenticate to external services.
+- [Issue and govern identities for automated
+  systems](machine-id/introduction.mdx) using Machine ID.
+
+### Teleport Identity
+
+**Teleport Identity** is an add-on to Teleport Access that offers identity
+governance and security for all of your infrastructure. You can provide users
+less-privileged roles by default, requiring any user who wants additional
+permissions to request them for a limited time. This approach leaves no
+permanent admin roles for attackers to target. 
+
+As an additional layer of protection against phishing and exfiltration, you can
+restrict access to users with trusted devices. And during active security
+incidents, you can lock down specific users, roles, infrastructure resources and
+more, letting you contain the security incident with minimal disruption. An
+audit log provides visibility into access patterns, so you can identify weak
+restrictions and potential security breaches.
+
+Get started with Teleport Identity:
+
+- [Access Requests](./access-controls/access-requests.mdx): Temporarily
+  provision minimal privileges to complete a task.
+- [Access Lists](./access-controls/access-lists.mdx): Regularly audit and
+  control membership to specific roles and traits, which then tie easily back
+  into Teleport's existing RBAC system.
+- [Device Trust](./access-controls/device-trust.mdx): Require an up-to-date,
+  registered device for each authentication by giving every device a
+  cryptographic identity. 
+- [Session & Identity Locks](./access-controls/guides/locking.mdx): Lock
+  suspicious or compromised identities and stop all their activity across all
+  protocols and services.
+- [Access Monitoring](./access-controls/access-monitoring.mdx): Detect overly
+  broad privileges and inspect sessions that are not using strong protection,
+  such as multi-factor authentication and Device Trust.
+
+### Teleport Policy
+
+**Teleport Policy** unifies and controls access policies across all of your
+infrastructure. With Teleport Access Graph, you can get insight into role-based
+access control policies, including in Teleport and your cloud provider. 
+
+Get started with [Teleport Access Graph](access-controls/access-graph.mdx).
+
+## Architecture
+
+The Teleport Access Platform consists of a certificate authority and
+identity-aware access proxy that run either on the Teleport-managed cloud or, in
+special cases, a self-hosted private network. 
+
+Teleport Agents, which the user deploys on Linux servers or Kubernetes, proxy
```

Comment:
```suggestion
Teleport Agents, which are deployed on Linux servers or Kubernetes clusters, proxy
```

Reply:
I've suggested "can run" to avoid raising questions of how the deployment works